opennms-model/src/main/java/org/opennms/netmgt/model/AlarmAssociation.java

```diff
@@ -45,6 +45,8 @@
 import javax.persistence.TemporalType;
 import javax.persistence.UniqueConstraint;
 
+import org.springframework.core.style.ToStringCreator;
```

Comment:
Can we please use google guava's https://google.github.io/guava/releases/19.0/api/docs/com/google/common/base/Objects.ToStringHelper.html here instead?

Reply:
fwiw, it's marked as deprecated. Did you want to use MoreObjects.ToStringHelper instead?
also fwiw, i brazenly copied this idiom from one of the other model classes (to be consistent) - there are about a dozen and a half that use ToStringCreator. Shall we (in a separate commit) migrate them all?

Reply:
Yeah, I meant `MoreObjects.ToStringHelper`, sorry.

My concern was just, to not introduce more Spring ToStringCreators, as it may require the spring core dependency vs the guava dependency, which may leads to problems if we want it to behave more nicely with osgi. 

Besides that we already use ToStringHelper (guava) and the apache commans equivalent. Until now I didn't know that there even is a ToStringCreator and we use it. But who was I kidding that we were not using it (-: Anyways as it is not used that heavily, I would opt for migrating them to guava instead.

Reply:
done. and did the rest under JIRA https://issues.opennms.org/browse/NMS-10455